lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -22,30 +21,175 @@ function distribute(node) {
+
+  const [radicand, root] = base.args;
+
+  const one = Node.Creator.constant(1);
+  const rootExponent = Node.Creator.operator('/', [one, root]);
+
+  const newExponent = root.equals(exponent)
+        ? one
+        : Node.Creator.parenthesis(Node.Creator.operator('*', [rootExponent, exponent]));
+
+  const newNode = Node.Creator.operator('^', [radicand, newExponent]);
+
+  return Node.Status.nodeChanged(
+    ChangeTypes.CONVERT_NTH_ROOT_TO_EXPONENT, node, newNode, false);
+}
+
+/*
+  Given a node raised to a negative power, negate the exponent and reciprocate
+  the node. Returns a nodeChanged Status object.
+
+  e.g. (x y)^-1 -> 1/(x y)
+  e.g. (2x^2)^-1 -> 1/(2x^2)
+*/
+function reciprocateTerm(node) {
+  const exponent = node.args[1];
+  // Negate the exponent
+  node.args[1] = Node.Type.isParenthesis(exponent)
+    ? Node.Creator.parenthesis(Negative.negate(exponent.content))
+    : Negative.negate(exponent);
+
+  const one = Node.Creator.constant(1);
+
+  const newNode = Node.Creator.operator('/', [one, node]);
+
+  return Node.Status.nodeChanged(
+    ChangeTypes.NEGATIVE_EXPONENT, node, newNode, false);
+}
+
+
+/*
+  Given an expression in the form (base)^exponent, where base is a multiplication node
+  and the exponent is any term, we want to distribute the exponent over each term.
+  Returns a nodeChanged Status object.
+
+  e.g. (x^2 y^2)^(2) -> x^4 y^4
+  e.g. ((x + 1) (x + 2))^2 -> (x + 1)^2 (x + 2)^2
+*/
+function distributeExponent(node) {
+  // Base is required to be a multiplication of terms
+  const base = getContent(node.args[0]);
+
+  // Exponent can be any term: polynomial, constant, fraction, etc
+  const outerExponent = getContent(node.args[1]);
+
+  let newTerms;
+
+  /*
+    When the base is a multiplication node
+    Iterate over each factor in the base and multiply the outside exponent
+    by the exponent of the term (implicit is one)
+    e.g. (x^2 y^2)^2 -> x^(2 * 2) y^(2 * 2)
+  */
+
+  if (Node.Type.isOperator(base, '*')) {
+    newTerms = base.args.map(function(term) {
+      // Recurses when a term is a multiplication node
+      // e.g. (2x^2 * y)^2 -> 2^2 * x^(2 * 2) * y^2
+      if (Node.Type.isOperator(term, '*')) {
+        const newTerm = Node.Creator.operator('^', [term, outerExponent]);
+
+        return distributeExponent(newTerm).newNode;
+      }
+
+      // When the base is a power node, multiply inner and outer exponents
+      // and wrap the new exponent in parens
+      // e.g. (x^2)^2 -> x^(2 * 2)
+      if (Node.Type.isOperator(term, '^')) {
+        return distributeOverPowerNode(term, outerExponent);
+      }
+
+      // e.g. (nthRoot(x,2) * nthRoot(x,3))^2 -> nthRoot(x, 2)^2 * nthRoot(x, 3)^2
+      if (Node.Type.isFunction(term, 'nthRoot')) {
+        return Node.Creator.operator('^', [term, outerExponent]);
+      }
+
+      return Node.Creator.operator('^', [term, outerExponent]);
+    });
+  }
+  else if (Node.Type.isOperator(base, '^')) {
```

Comment:
As per my other comment, it's a little confusing to have this case stuck at the bottom here. I'd have a different function handle the power case. It's okay if it shares a bit of code with the multiplication case.